README.md

```diff
@@ -224,7 +224,44 @@ end
 The above modification will allow your job to retry up to 10 times, with a delay
 of 120 seconds, or 2 minutes between retry attempts.
 
-You can override the `retry_delay` method to set the delay value dynamically.
+You can override the `retry_delay` method to set the delay value dynamically. For example:
+
+```ruby
+class ExampleJob
+  extend Resque::Plugins::Retry
+  @queue = :testing
+
+  def self.retry_delay(exception)
+    if exception == SocketError
+      10
+    else
+      1
+    end
+  end
+
+  def self.perform(*args)
+    heavy_lifting
+  end
+end
+```
+
+Or, if you'd like the delay to be dependent on job arguments:
+
+```ruby
+class ExampleJob
+  extend Resque::Plugins::Retry
+  @queue = :testing
+
+  def self.retry_delay(exception, *args)
+    # make sure an integer is returned
+    args.first.to_i
```

Comment:
This will return `0` by _default_ is that OK?

Reply:
@jzaleski the idea here it the delay is determined by the args to the job—depending on what is passed as the job args the delay could be different. This is not clear in the ex; will update.